dwave_networkx/generators/pegasus.py

```diff
@@ -256,24 +260,39 @@ def efilter(e): return qfilter(*e[0]) and qfilter(*e[1])
 
     if data:
         v = 0
+        def coord_label():
+            return q
+        def int_label():
+            return v
+        if nice_coordinates:
+            def this_label():
+                return pegasus_to_nice(q)
+            other_name = 'linear_index'
```

Comment:
nice_index?

Reply:
Sorry, yeah, this code is quite annoying.  I tried to homogenize the approach behind filling in the labels... I think I can do better.

Reply:
But, no, not nice_index -- `other_name` corresponds to the name of the field that's being filled in by `other_label`